Missing build dependency: PHP

### Description

PHP is a build dependency.

#### How to reproduce
On a clean Debian Jessie:
```
$ apt-get update
$ apt-get install -y --no-install-recommends git lsb-release sudo
$ ./utils/fo-installdeps --buildtime -y
$ make build-cli
[...]
Generating fo_wrapper ...
make[2]: ../../utils/preprocess_php: Command not found
../../Makefile.process:24: recipe for target 'fo_wrapper-process_php' failed
make[2]: *** [fo_wrapper-process_php] Error 127
make[2]: Leaving directory '/fossology/src/cli'
Makefile:51: recipe for target 'build-cli' failed
make[1]: *** [build-cli] Error 2
make[1]: Leaving directory '/fossology/src'
Makefile:27: recipe for target 'build-src' failed
make: *** [build-src] Error 2
```

Similar error for the following targets:
build-cli
build-decider
build-deciderjob
build-readmeoss
build-reportImport
build-reuser
build-spdx2
build-unifiedreport

--- a/utils/fo-installdeps
+++ b/utils/fo-installdeps
@@ -104,14 +104,16 @@ if [[ $BUILDTIME ]]; then
             libmxml-dev curl libxml2-dev libcunit1-dev \
             build-essential libtext-template-perl subversion rpm librpm-dev libmagic-dev libglib2.0 libboost-regex-dev libboost-program-options-dev
          case "$CODENAME" in
+           jessie)
+             apt-get $YesOpt install php5-cli;;
            xenial|stretch)
-             apt-get $YesOpt install php-mbstring php7.0-xml php7.0-zip;;
+             apt-get $YesOpt install php-mbstring php7.0-cli php7.0-xml php7.0-zip;;
            buster)
-             apt-get $YesOpt install php-mbstring php7.3-xml php7.3-zip;;
+             apt-get $YesOpt install php-mbstring php7.3-cli php7.3-xml php7.3-zip;;
            bionic)
-             apt-get $YesOpt install php-mbstring php7.2-xml php7.2-zip;;
+             apt-get $YesOpt install php-mbstring php7.2-cli php7.2-xml php7.2-zip;;
            sid)
-             apt-get $YesOpt install php-mbstring php-xml php-zip;;
+             apt-get $YesOpt install php-mbstring php-cli php-xml php-zip;;
          esac
          if ! dpkg --get-selections | grep -q postgresql-server-dev; then  ## if postgresql-server-dev is not installed
            case "$CODENAME" in

